### ISSUE TITLE
Lesson Builder duplicate name modal bug when saving draft/submitting

### ISSUE THREAD

**Issue by arielkennan** [codeforamerica/bizfriendly-web#162]
Steps to recreate: 
- Create a new lesson, don't edit the name
- Add some elements 
- Click Save as Draft or Submit 
- Get "A lesson with that name already exists." error msg. 
- Edit name
- Save/Submit again, but no modal appears (aka you think nothing happened)
- Click Save/Submit again, get "A lesson with that name already exists." error msg again as your lesson did indeed go through (visible in your profile). 

Basically the functionality of submitting/saving is working correctly, but the user feedback loop through the modal is failing.

**Comment by arielkennan** [comment]
I'm having problems with getting the modal even without getting the "A lesson with that name already exists." Seems like the modal trigger isn't consistently working on Saving Draft / Submitting.

**Comment by arielkennan** [comment]
I'm not getting modals at all on staging upon submitting/saving drafts of lesson. I've refreshed a bunch of times and seeing all other updates. I get modals just fine for skills/services.

**Comment by ondrae** [comment]
Check it now. Should be fixed.
